atom/browser/mac/mac_native_converter.mm

```diff
@@ -0,0 +1,54 @@
+#import "atom/browser/mac/mac_native_converter.h"
+
+@implementation MacNativeConverter
```

Comment:
This is getting a little out of hand and I feel like I must be duplicating code that already exists somewhere.
@zcbenz: Is there a better way to convert between Cocoa Foundation types and `base` types?